0000-code-name-support.md

```diff
@@ -0,0 +1,116 @@
+- Feature Name: code-name-support
+- Start Date: 2024-09-04
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Add optional code_name parameter to Cargo.toml [package] section.
+
+# Motivation
+[motivation]: #motivation
+
+Many software projects use code names for their releases. It allows you the possibility
+to refer to a program not only by its version number but also by code name or marking name.
+
+For example instead of Windows 4.0.950 C people usually say Windows 95.
```

Comment:
Windows 95 might also refer to v4.0.951 and v4.1.234 though. I guess it's more like a grouping of versions, not a singular version.

Reply:
Yes, that is why I wrote "and for each major or minor release" - depends on the convention that you want to use.